custom/succeed/models.py

```diff
@@ -107,4 +107,43 @@ class UCLAPatientFluff(fluff.IndicatorDocument):
 
     emitter = RandomizationDate()
 
+
+def get_full_name(case):
+    try:
+        return CommCareCase.get(case['indices'][0]['referenced_id'])['full_name']
+    except AttributeError:
+        return None
+
+
+class Emitter(fluff.Calculator):
+
+    @fluff.null_emitter
+    def date(self, case):
+        yield None
+
+
+class UCLATaskActivityFluff(fluff.IndicatorDocument):
```

Comment:
Is this for getting the 'referenced_id' field out of a case index? I think @czue is better equipped to answer that but I don't think we have a way of doing it at the moment. @czue do we want to add in json path?